### ISSUE TITLE
block: preset and template options should be simplified & reduce duplication

### ISSUE THREAD

**Issue by NovemLinguae** [wikimedia-gadgets/twinkle#1987]
The current block interface seems complicated.

![image](https://github.com/user-attachments/assets/623d5456-304e-4610-a250-b5cd74398a39)

For example, I often end up typing the block reason twice. Once for the template's "you have been blocked for" field, and once for the block log. In my opinion, you should only have to type it once, and then Twinkle handles placing it in both the template and the block log.

Also, do we really need to pick from so many block templates? Isn't there just a couple generic ones (regular block, AE block), and our typed block reason can be passed to it as a parameter?

This ticket is to explore either simplifying the block module, or adding a mode that is simpler than the current mode used.

MediaWiki's Special:Block is simple but doesn't place templates. Twinkle's block module places templates but is complicated. The ideal is probably somewhere between this. We need Twinkle to help with placing a block template, but the form itself should be easy to use.

**Comment by voorts** [comment]
I agree that the format is overcomplicated, but I like the variety of specific block templates and would like to see an option that lets you add custom block templates, similar to the feature for maintenance tags.

**Comment by NovemLinguae** [comment]
Sounds like a different feature, so I filed #2092. These two tickets do not seem mutually exclusive :)
